app/middleware/getGp.js

```diff
@@ -22,7 +22,7 @@ function getGp(req, res, next) {
       next();
     });
   }).catch((err) => {
-    log.error(err.stack, `Error connecting to ${connectionString}`);
+    log.error(err, `Error connecting to ${connectionString}`);
```

Comment:
Small detail but the bunyan docs suggest this should be of the form `log.error(err, 'Error connecting to %s', connectionString)`